packages/@sfx/core/test/unit/common/utils/core.test.ts

```diff
@@ -476,5 +476,33 @@ describe('CoreUtils', () => {
 
       expect(() => unregisterPlugins(names, registry, directory)).to.not.throw();
     });
+
+    it('should ignore plugins not in the directory', () => {
```

Comment:
This is a bit of an odd test. I guess you're asserting that the function acts as normal if non-existent plugins are passed. Is there something simpler we can assert? Eg. that it doesn't throw an error? This is fine, but it looks like it's pretty much the same as the test above. Or maybe we can just wrap them into one test.

Reply:
It's fine and is not worth spending too much time.